src/app/templates/TokenListItem.tsx

```diff
@@ -121,12 +122,14 @@ interface EvmTokenListItemProps {
   assetSlug: string;
   manageActive?: boolean;
   onClick?: MouseEventHandler<TokenListItemElement>;
+  useIsVisible?: boolean;
```

Comment:
In my opinion, it's a bad practice to start the name of anything but for a hook with `use`